pkg/controller/kubedirectorcluster/members.go

```diff
@@ -401,6 +493,23 @@ func handleCreatingMembers(
 				m.StateDetail.ConfigErrorDetail = errorDetail
 			}
 
+			connectionVersion := int64(0)
+			if v, ok := cr.Annotations[shared.ConnectionsIncrementor]; ok {
+				connectionVersion, _ = strconv.ParseInt(v, 10, 64)
```

Comment:
what is the behavior if the annotation is accidentally deleted or update to non string

Reply:
We now have a fall-back mechanism to assign the minimum of the connection version from the ready members.